Firestore/Example/Tests/Model/FSTFieldValueTests.mm

```diff
@@ -17,8 +17,8 @@
 #import "Firestore/Source/Model/FSTFieldValue.h"
 
 #import <FirebaseFirestore/FIRGeoPoint.h>
-#import <FirebaseFirestore/FIRTimestamp.h>
 #import <XCTest/XCTest.h>
+#import "FIRTimestamp.h"
```

Comment:
From tests these should be framework imports, like `#import <FirebaseFirestore/FIRGeoPoint.h>` above.

Reply:
Changed in all test files.